code/src/index.css

```diff
@@ -1,13 +1,231 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
-}
+* {
+	box-sizing: border-box;
+}
+
+code {
+	font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+		monospace;
+}
+
+body {
+	margin: 0;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+	background-image: url("/images/lights.jpg");
+	color: #fff;
+	display: flex;
+	justify-content: center;
+	backdrop-filter: brightness(0.4);
+	font-family: "Karla", sans-serif;
+}
+
+.root {
+	display: flex;
+	align-items: center;
+}
+
+.app-container {
+	display: flex;
+	width: 80vw;
+	max-width: 310px;
+	height: 100vh;
+	align-items: center;
+	justify-content: center;
+}
+
+.welcome-header,
+.summary-header {
+	font-size: 25px;
+	font-family: "Rubik", sans-serif;
+	margin-bottom: 50px;
+}
+
+.question-container {
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+	width: 100%;
+	min-height: 250px;
+	align-self: flex-start;
+}
+
+.progress-bar {
+	align-self: flex-end;
+	display: flex;
+	align-items: center;
+	margin-top: 40px;
+}
+
+.progress {
+	height: 25px;
+	width: 282px;
+	border-radius: 0;
+	border-bottom: 1px solid #a9a9a9;
+}
+
+progress::-webkit-progress-bar {
+	background-color: transparent;
+}
+
+progress::-webkit-progress-value {
+	background-color: #fff;
+	opacity: 0.3;
+}
+
+output {
+	font-size: 12px;

```

Comment:
It is worth considering that a larger font size on desktop will be easier to read for visually impaired users. (Recommended 16px and above)